src/main/java/com/google/sps/servlets/NewsServlet.java

```diff
@@ -40,9 +40,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     HashMap<String, List<Article>> articleMap = new HashMap<>();
     terms.forEach((term) -> {
       try {
+        System.out.println("In try loop");
         Map<String, String> paramMap = new HashMap<>();
         paramMap.put("q", UrlRequest.encodeTerm(term));
-        paramMap.put("gl", getCountry(request));
+        System.out.println("after encode term");
+        //paramMap.put("gl", getCountry(request));
```

Comment:
Is this supposed to be commented out?

Reply:
Yes -- this param is what integrates location to the Google News query. Because we are no longer using cookies to store the coordinates, this process will need to be changed, and I am waiting on a PR that modifies load.js before getting started on this problem.

Reply:
[Optional] I would add a TODO comment explaining it.

Reply:
Yes, I would either remove this entirely or add a TODO with details indicating why it needs to left in as a comment.